src/components/Home/CurrEvents.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import '../../styles/Events.css';
+import '../../styles/Home.css';
+import hfamimg from '../../images/hack-fam-insta.png';
+import schoolimg from '../../images/hack-school-insta.jpg';
```

Comment:
Rename `hfamimg` and `schoolimg` to something along the lines of "img1" and "img2" respectively-- this is just so there's more reusability in the future.